core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/KeyStoreManager.java

```diff
@@ -118,99 +127,91 @@ public static KeyStoreManager getInstance(int tenantId,
     }
 
     /**
-     * Get the key store object for the given key store name
+     * Get the key store object for the given key store name.
      *
-     * @param keyStoreName key store name
-     * @return KeyStore object
-     * @throws Exception If there is not a key store with the given name
+     * @param keyStoreName  Key store name.
+     * @return KeyStore object.
+     * @throws Exception    If there is an error when retriving the given keystore.
      */
     public KeyStore getKeyStore(String keyStoreName) throws Exception {
 
+        if (keyStoreName == null || keyStoreName.isEmpty()) {
```

Comment:
```suggestion
        if (StringUtils.isEmpty(keyStoreName)) {
```

Reply:
Fixed in https://github.com/wso2/carbon-kernel/pull/4025/commits/9584f218d7768dec546d2b31108c345036396e6a